### PR TITLE
Adding code module for standup module

### PR DESCRIPTION
**Pull request by thomastk**
##### SUMMARY

Adding a new module standup, that can be used to verify and monitor a group of machines that would serve multiple application roles in a software system.

##### ISSUE TYPE

 - New Module Pull Request
##### COMPONENT NAME

standup
##### ANSIBLE VERSION

```
ansible 2.2.1.0
  config file = 
  configured module search path = Default w/o overrides
```
##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

File: lib/ansible/modules/monitoring/standup.py [hunk 1]
```diff
@@ -0,0 +1,533 @@
...
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+author: "Thomas K. Theakanath (thomastk@"
+module: standup
+short_description: Runs a suite of checks to validate a host and heals state if needed.
+description:
+    - "The standup moudle runs a set of commands and evaluate the output to determine if a host stands up right. The tests can be specified in YAML format. A check contains a command that is supported on the host and directives to evaluate its output. An optional command to heal a state can also be specified with every check."
```
Comment (mscherer) [inline lib/ansible/modules/monitoring/standup.py]: s/moudle/module/

File: lib/ansible/modules/monitoring/standup.py [hunk 2]
```diff
@@ -0,0 +1,533 @@
...
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this software.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['stableinterface'],
```
Comment (s-hertel) [inline lib/ansible/modules/monitoring/standup.py]: This should be 'preview' status probably?

File: lib/ansible/modules/monitoring/standup.py (hunk 1 above)
Comment (thomastk) [reply lib/ansible/modules/monitoring/standup.py]: Thanks for locating the typo.

File: lib/ansible/modules/monitoring/standup.py (hunk 2 above)
Comment (thomastk) [reply lib/ansible/modules/monitoring/standup.py]: The interface to the module is not expected to change as this module is an Ansible version of an infra verification tool that I had developed earlier. But module is developed from scratch and yet to be used extensively in the field. Would that make the status of the module "preview"?

**Comment by thomastk** [comment]
Will add another pull request after checking in fixes.

**Pull request closed without merging**